TODO.md

```diff
@@ -7,6 +7,18 @@ Scratchpad for random ideas of things that need done.
 * Verify RDP enable
 * Linux repos
 * Whole buncha stuff for profile::cfcc::camper
+Charlie's $0.02
+
+* jdk8 > jdk10
+* remove eclipse
+* add a good text editor (I use notepad2-mod; I think lots of the others are overkill and overwhelming, nbd though)
```

Comment:
that's what notepad++ is for? https://github.com/cohoe/TurboPuppet/blob/production/manifests/profile/tools/notepadplusplus.pp

I also have one in for Sublime which is more cross-platform.

Reply:
"something something vscode"

Reply:
Ohhhh, I didn't see the inheritance and profiles; now I need to make more comments on those! VS Code is bloated electron garbage ;)

Reply:
Most of the interesting bits happen in https://github.com/cohoe/TurboPuppet/blob/production/manifests/profile/cfcc/camper.pp